keyvalue/clients/src/main/java/tachyon/client/keyvalue/hadoop/KeyValueInputFormat.java

```diff
@@ -67,8 +67,9 @@
     List<InputSplit> splits = Lists.newArrayList();
     try {
       for (Path path : paths) {
-        List<PartitionInfo> partitionInfos =
-            mKeyValueMasterClient.getPartitionInfo(new TachyonURI(path.toString()));
+        TachyonURI tachyonURI = new TachyonURI(path.toString());
```

Comment:
comparing to `FileSystemMasterClient`, `getPath` should be used in `KeyValueMasterClient` in stead of `toString` for `TachyonURI`. So, yes, it's ugly to drop scheme and authority here.